src/components/resource/edit-managed-title/resource-edit-managed-title.js

```diff
@@ -162,26 +162,38 @@ class ResourceEditManagedTitle extends Component {
               <DetailsViewSection
```

Comment:
On the full page edit page - we should Selected/Hidden below Holding Status and Above Toggle - should we have it here as well to be consistent?